ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/XARecoveryModuleUnitTest.java

```diff
@@ -374,6 +378,106 @@ public void start(Xid xid, int i) throws XAException {
         assertEquals(recoverCalled, 6);
     }
 
+    @Test
```

Comment:
We don't actually have orphan detection for JTS: https://issues.jboss.org/browse/JBTM-2124